tests/test_text_generation_example.py

```diff
@@ -55,7 +56,10 @@
             ("baichuan-inc/Baichuan2-7B-Chat", 1, True, 108, False),
             ("baichuan-inc/Baichuan2-13B-Chat", 1, False, 66, False),
             ("deepseek-ai/DeepSeek-V2-Lite", 1, False, 35, False),
+            ("THUDM/chatglm2-6b", 1, True, 150, False),
             ("THUDM/chatglm3-6b", 1, True, 150, False),
+            ("Qwen/Qwen2.5-7B", 4, False, 490, False),
+            ("Qwen/Qwen2.5-32B", 4, False, 120, False),
```

Comment:
can you remove 7B from multi test?

Reply:
Do you mean remove "Qwen/Qwen2.5-7B"? I think we need to test small models. Customers use small models to test and build their APPs.